cmd/gen-doc/gen-doc.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"path"
+	"strings"
+
+	"github.com/koyeb/koyeb-cli/pkg/koyeb"
+	"github.com/spf13/cobra/doc"
+)
+
+func genMarkdownDocumentation() {
+	rootCmd := koyeb.GetRootCmd()
+	linkHandler := func(name string) string {
+		base := strings.TrimSuffix(name, path.Ext(name))
+		return "#" + strings.Replace(strings.ToLower(base), "_", "-", -1)
+	}
+
+	filePrepender := func(filename string) string {
+		fmt.Printf(filename)
+		if filename == "docs/koyeb.md" {
+			return `---
+title: "Koyeb CLI Reference"
+shortTitle: Reference
+description: "Discover all the commands available via the Koyeb CLI and how to use them to interact with the Koyeb serverless platform directly from the terminal."
+---
+
+The Koyeb CLI allows you to interact with Koyeb directly from the terminal. This documentation references all commands and options available in the CLI.
+
+If you do not have installed the Koyeb CLI yet, please read the [installation instructions guide](/docs/cli/installation).
+`
+		}
+		return ""
+	}
+
+	rootCmd.DisableAutoGenTag = true
```

Comment:
Can you move it to the construction of the rootcmd directly ?

Reply:
Better like this?

Reply:
yep